### DISCUSSION TITLE
No design on standard easy admin implementation

### DISCUSSION THREAD

**Discussion by chris576** [symfony/symfony#50989] [Q&A]
Hey 👍 

I am trying to implement a standard admin panle, using easy admin 4. I works fine, but I just can see this: 

![grafik](https://github.com/symfony/symfony/assets/59051160/1f43a242-b230-4653-856a-89f822e355a7)

Due to the Documentation ( https://symfony.com/doc/6.2/the-fast-track/de/9-backend.html ) it should look like this: 

![grafik](https://github.com/symfony/symfony/assets/59051160/9c7962f0-4ce7-4cf5-8961-7892f0c062f6)

The code can be inspected here: https://gitlab.technik-emden.de/ch6447/HardwareBookingSystem/-/tree/admin_interface?ref_type=heads

**Comment by alexisLefebvre** [comment]
Did you run `php bin/console assets:install`? See [EasyAdminBundle doesn't load assets](https://stackoverflow.com/questions/54158995/easyadminbundle-doesnt-load-assets).

**Reply by chris576** [reply]
I am trying to run the application within a docker-container: 

```
FROM php:8.1-fpm AS composer

RUN apt-get update && apt-get install -y \
    unzip \
    libicu-dev \
    libonig-dev \
    libzip-dev

RUN docker-php-ext-install \
    intl \
    zip \
    pdo_mysql

WORKDIR /var/www/html/

COPY . .

RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer

RUN composer install

RUN composer dump-autoload --optimize

RUN chown -R www-data:www-data ./var

FROM node:latest AS node

WORKDIR /var/www/html/

COPY --from=composer /var/www/html/ .

RUN yarn install

RUN yarn encore dev

FROM composer

WORKDIR /var/www/html/

COPY --from=node /var/www/html/ .

RUN php bin/console assets:install

CMD ["php-fpm"]
```

Running `php bin/console assets:install` does not make change.

**Reply by chris576** [reply]
I wonder, that the default page is shown correctly, while the asssets for the crud are not loaded:

![grafik](https://github.com/symfony/symfony/assets/59051160/c8d90a36-4c5c-4ab9-abc4-46a814a1ff39)

**Reply by alexisLefebvre** [reply]
Is it possible that the 3 `COPY` overwrite the files? Do you find the assets files in the `public` directory? (I mean the CSS and JS files that should be loaded by your browser)
